google/auth/crypt.py

```diff
@@ -229,6 +231,29 @@ def from_string(cls, key, key_id=None):
         return cls(private_key, key_id=key_id)
 
     @classmethod
+    def from_service_account_info(cls, info):
+        """Creates a Signer instance instance from a dictionary containing
+        service account info in Google format.
+
+        Args:
+            info (Mapping[str, str]): The service account info in Google
+                format.
+
+        Returns:
+            Signer: The constructed signer.
+
+        Raises:
+            ValueError: If the info is not in the expected format.
+        """
+        if 'private_key' not in info:
```

Comment:
Make `'private_key'` a module-level non-public constant? If for no other reason than being about to refer to it in tests.